### ISSUE TITLE
New graphic interface

### ISSUE THREAD

**Issue by cosme12** [cosme12/cheatsheet-maker#11]
Quote from reddit user u/animalAbit
> Cool Project. However, the CLI interface not my favorite for this case. I would love something like editing a text file in some markdown like format and the tool would produce a neatly formatted html cheat sheet(or any desirable format) just like your final output. Something akin to jekyll, but for cheat sheets. That way, I can edit freely and also choose to reuse my work if I choose to.

**Comment by srcecde** [comment]
How about creating a interface with PyQt5?

**Comment by cosme12** [comment]
That's the plan. But first I want to finish the basic stuff like the exporter first before doing a big version jump.

**Comment by cosme12** [comment]
@srcecde @JCharante any idea of how we should build the interface? I have never worked with pyQt before. Should the html webview be in the same ui window or different?

**Comment by srcecde** [comment]
Well, we can generate the preview in same window as well as in browser. But it depends on the number of components in GUI to be displayed. Because i think we don't want to create a messy and congested  layout.

**Comment by cosme12** [comment]
Hmm... right.

**Comment by JCharante** [comment]
In regards to the preview, I like [Gummi's](https://en.wikipedia.org/wiki/Gummi_(software)) editor style where they have your markup on the left, and the live preview on the right.

For the stack, after looking at some quick comparisons with PyGTK I feel that going with PyQt is the right choice. I've also never used it before, but I could certainly expand my knowledge in creating gui outside of the web ecosphere so if time permits I'll definitely try to help out with that.

**Comment by cosme12** [comment]
Yep, we should include sheet preview here too. That Gummi's ui seems like a cool start. As soon as I get some free time I will reasearch how pyQt5 works and do some mock ups.

**Comment by cosme12** [comment]
Also, why @srcecde is not shown as contributor? You pushed several pull request...

**Comment by JCharante** [comment]
@cosme12 None of @srcecde's changes have made it to the `master` branch, they're still in the `develop` branch. He'll show up as a contributor in the next release.
